ROOTFS/etc/sage/install.sh

```diff
@@ -0,0 +1,864 @@
+#!/bin/sh
```

Comment:
can we name this file `k3s_install.sh`?

Also please move to `/etc/waggle/` 

we are retiring the `/etc/sage/` folder and moving everything to `/etc/waggle/` to reduce confusion and to more easily support legacy tools.